easybuild/easyconfigs/b/binutils/binutils-2.26_GCC-6.patch

```diff
@@ -0,0 +1,29 @@
+Add unused attribute to Dir_caches to avoid error with GCC 6 which treats warnings as errors. See https://sourceware.org/bugzilla/show_bug.cgi?format=multiple&id=19751
+by Davide Vanzo (Vanderbilt University)
```

Comment:
Not really by you, taken from upstream bug (see included URL), so just mention the URL?

Reply:
I added the bug URL in the file. I had to modify the original patch because patching the changelog file failed.

Reply:
OK, makes sense, thanks for clarifying.